pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -38,10 +39,11 @@ import (
 const (
 	karpenterManagedTagKey = "karpenter.azure.com/cluster"
 
-	vnetDataPlaneLabel      = "kubernetes.azure.com/ebpf-dataplane"
-	vnetSubnetNameLabel     = "kubernetes.azure.com/network-subnet"
-	vnetGUIDLabel           = "kubernetes.azure.com/nodenetwork-vnetguid"
-	vnetPodNetworkTypeLabel = "kubernetes.azure.com/podnetwork-type"
+	vnetDataPlaneLabel       = "kubernetes.azure.com/ebpf-dataplane"
+	vnetAzureCNIOverlayLabel = "kubernetes.azure.com/azure-cni-overlay"
```

Comment:
Why do these constants have 'vnet' prefix?

Reply:
Good question, followed the naming pattern but was wondering the same thing. I'm going to remove the prefix.